card/mod/05_email/set/right/following.rb

```diff
@@ -13,21 +13,40 @@ def virtual?; true end
      end
    end
    
+   view :status do |args|
+     if rcard = current_follow_rule_card
+       rcard.item_cards.map do |item|
+         %{<div class="alert alert-success" role="alert">
+           <strong>#{item.title}:</strong> #{rcard.rule_set.follow_label}
+          </div>}
+       end * ', '
+     else
+       "No following preference"
+     end
+   end
+   
```

Comment:
basic idea is to make it so that there's a simple view where you can see what you're following.  Made it so that you go there after editing the rule.